ci/run_black.ps1

```diff
@@ -0,0 +1,19 @@
+$ExitCode = 0
+$GitDiff = git diff origin/master
+# If there is no diff between master, then black everything.
+if ( $GitDiff -eq $null ) {
+    black .
+    if ($LastExitCode -ne 0) { $ExitCode = $LastExitCode }
+}
+# Else black just the diff.
+else {
+    Write-Output $GitDiff | black
```

Comment:
The purpose of this statement is to ONLY run black on files that changed in this PR compared to master, not on the entire project. `flake8` had a `--diff` flag that would read the `git diff` as input and only check the diff, instead of checking the entire project. Can you confirm that this line actually works as expected? If not, a close approximation would be to read the filenames returned from `git diff` and only run black on those filenames, instead of on the entire directory.